frontend/src/Onboarding/LoginScreen.ts

```diff
@@ -0,0 +1,350 @@
+export {};
```

Comment:
Going to comment this whole file out for now, as we won't have to worry about it (until we expand to other colleges)